repos/libports/src/lib/ada/runtime/a-except.adb

```diff
@@ -1,3 +1,13 @@
+--
+--  \brief  Ada exceptions
+--  \author Johannes Kliemann
+--  \date   2018-06-25
+--
+--  Copyright (C) 2018 Componolit GmbH
+--  This file is part of the Genode OS framework, which is distributed
```

Comment:
missing newline below copyright line

Reply:
I wonder, would you mind also adding Genode Labs as copyright holder since the changes are contributed under the GCA. I want to make it clear that Genode Labs is in the legal position to license Genode - including these files - commercially. @senier What do you think?

Reply:
No objections. There is [at least one](https://github.com/genodelabs/genode/blob/master/repos/libports/src/lib/pthread/rwlock.cc) other instance where we did not do this. We can also update those other occurences with a Genode copyright header if that helps clarifying things to your customers.